music_assistant_models/smart_fades.py

```diff
@@ -0,0 +1,30 @@
+"""Model(s) for smart fades analysis."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+import numpy as np
```

Comment:
Not sure if I'm a fan of having a super heavy dependency such as numpy to the models package.
It also makes you wonder if the smart fades data should be exposed at all ?
Cant we keep this server side only ?

Reply:
Good point. My idea was that we might eventually want to expose certain bits of it so the user can see things like 'analyzed bpm' in the UI. Or just like @jozefKruszynski suggested, whether a Track is 'ready for smart fades'.

Reply:
^ This implies we cannot add it to StreamDetails either, as this would create a dependency between models > server.